upup/models/cloudup/resources/addons/spotinst-kubernetes-cluster-controller.addons.k8s.io/v1.8.0.yaml.template

```diff
@@ -84,33 +97,47 @@ apiVersion: apps/v1beta1
 kind: Deployment
 metadata:
   labels:
-    k8s-addon: spotinst-kubernetes-cluster-controller.addons.k8s.io
+    k8s-app: spotinst-kubernetes-cluster-controller
   name: spotinst-kubernetes-cluster-controller
   namespace: kube-system
 spec:
   replicas: 1
   revisionHistoryLimit: 10
   selector:
     matchLabels:
-      k8s-addon: spotinst-kubernetes-cluster-controller.addons.k8s.io
+      k8s-app: spotinst-kubernetes-cluster-controller
```

Comment:
Unfortunately this field isnt mutable, so updating the Deployment resource will fail. I think we should keep the `k8s-addon` label on all the resources but update everything else to match the upstream manifest :( it is not ideal.